engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/manifest/manifest-import.controller.js

```diff
@@ -248,5 +249,13 @@ angular.module('Bastion.subscriptions').controller('ManifestImportController',
                 $scope.showHistoryMoreLink = $scope.isTruncated($scope.statuses, results);
             });
         });
+
+        $scope.manifestRefreshDisabled = function () {
+          return $scope.isTaskPending() ||
+                 $scope.upstream == undefined ||
+                 $scope.upstream.idCert == undefined ||
+                 $scope.upstream.idCert.cert == undefined ||
```

Comment:
You may want to use `===` or there's an `_.isUndefined()` if you prefer that.

Will these ever be false or null?  If so does it make sense to just use `!$scope.upstream.idCert` etc.

Reply:
I was thinking the same thing, I was copying the syntax from the html, I am not sure if there was an original reason for comparing to undefined exclusively. I'm honestly not even sure how to recreate these conditions, but I'll poke around a bit

I can definitely update to `===`, I don't see a reason not to do that

Reply:
@waldenraines I checked, `$scope.upstream.idCert.cert` is literally a certificate, It should be fine to use `!$scope.upstream.idCert`, I updated to use that